test/Ocelot.AcceptanceTests/HttpTests.cs

```diff
@@ -103,7 +103,7 @@ public void should_return_response_200_when_using_http_two_point_zero()
                         DownstreamPathTemplate = "/{url}",
                         DownstreamScheme = "https",
                         UpstreamPathTemplate = "/{url}",
-                        UpstreamHttpMethod = new List<string> { "Get" },
+                        UpstreamHttpMethod = new List<string> { "Put" },

```

Comment:
What are doing, man? @abriening 
Current test should remain unchanged because it was written for GET case.